tests/Unit/Domain/Creators/Test_BinaryCompactObject.cpp

```diff
@@ -39,14 +38,55 @@ class FunctionOfTime;
 }  // namespace domain::FunctionsOfTime
 
 namespace {
-using Translation = domain::CoordinateMaps::TimeDependent::Translation;
-using Translation3D = domain::CoordinateMaps::TimeDependent::ProductOf3Maps<
-    Translation, Translation, Translation>;
 using BoundaryCondVector = std::vector<DirectionMap<
     3, std::unique_ptr<domain::BoundaryConditions::BoundaryCondition>>>;
 using Object = domain::creators::BinaryCompactObject::Object;
 using Excision = domain::creators::BinaryCompactObject::Excision;
 
+template <size_t Dim>
+struct TimeIndependentMetavariablesWithBoundaryCondition {
```

Comment:
(optional) I think you can just add a `bool TimeDependent` template parameter to this struct, delete the `TimeIndependent` from the name and assign `enable_time_dependent_maps = TimeDependent` below. Then you only need 2 instead of 4 structs. You could even get down to a single struct with another template parameter `UseBoundaryConditions` and `using system = tmpl::conditional_t<...>`.

Reply:
I replaced this with just one struct